nicegui/elements/qtable.js

```diff
@@ -0,0 +1,75 @@
+function eval_fields(cols) {
+    // For each Field, Sort and Format field within the column definition,
+    // look for '=>' and if so, try to eval to run the function.
+
+    cols.forEach((col) => {
+        if (!col.field || !col.field.includes("=>")) return;
+        col.field = eval(col.field);
+
+        if (!col.sort || !col.sort.includes("=>")) return;
+        col.sort = eval(col.sort);
+
+        if (!col.format || !col.format.includes("=>")) return;
+        col.format = eval(col.format);
+    });
+}
+
+export default {
+    template: `
+        <q-table
+            :rows="rows"
+            :columns="columns"
+
+            :row-key="this.selection_key"
+            :selection="this.selection_mode"
+
+            v-model:selected="selected"
+            @selection="onSelection"
+
+            :filter="filter"
+        >
+
+        <template v-slot:top-right>
```

Comment:
Could I suggested to always use the v-slot body syntax ? 
https://quasar.dev/vue-components/table#body-slots

That would be more complex in that PR but allow much more flexibilty. Without giving too much thoughts, maybe something like : 
```
with ui.table():
  with ui.tablerow():
     with ui.tablecolumn('name'):
        ui.label('Here we can have custom elements')
     with ui.tablecolumn('actions'):
        # Here maybe a dropdown menu with 'edit' / 'delete' icons on the row for instance
```